test/NuGet.Core.Tests/NuGet.CommandLine.Xplat.Tests/XPlatConfigUnitTests.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.CommandLine.XPlat;
+using Xunit;
+
+namespace NuGet.CommandLine.Xplat.Tests
+{
+    public class XPlatConfigUnitTests
+    {
+        [Theory]
+        [InlineData(true, false)]
+        [InlineData(false, true)]
+        public void ConfigPathsCommand_WithNullArguments_ThrowsArgumentNullException(bool argsIsNull, bool loggerIsNull )
+        {
+            // Arrange
+            ConfigPathsArgs args = null;
+            CommandOutputLogger getLogger = null;
+
+            if ( !argsIsNull )
```

Comment:
nit: remove extra spaces. If you use Ctrl + K + D in Visual Studio, it should format your code for you and remove these spaces. There are other occurrences of spacing around parentheses, but I'll just leave this comment for all of them. If you use the format feature, it should clean them all up.

```suggestion
            if (!argsIsNull)
```